csrc/scheduler/vectorize_helper.h

```diff
@@ -314,6 +314,14 @@ int64_t getVectorizationFactor(
     HeuristicSummary* data_cache,
     int64_t break_point);
 
+int64_t getVectorizationVectorTransposeGroup(
```

Comment:
nit: we usually call it a factor, so `getVectorizationFactorTransposeGroup` would be more consistent